### PR TITLE
Updated fixed-reported-inaccuracies with Aug & Sep fixes

### PR DESCRIPTION
**Pull request by v-Jagadeeshl**

### CODE REVIEW

File: microsoft-365/security/defender-vulnerability-management/fixed-reported-inaccuracies.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ The report inaccuracy capabilities provides a way to report false positive, inac
...
 You can use this article to find details on the inaccuracies that have been reported and are now fixed. The following tables present the fixed inaccuracies organized by month:
 
+## September 2023
+
+Inaccuracy report ID |Description |Fix date |
+|:---|:---|:---|
+| - |Added accuracte EOS details for Redis| 04-Sep-23
+| 31688 |Fixed inaccuracy in CVE-2023-38831 and CVE-2023-40477| 05-Sep-23
+| 31898 |Fixed Inaccuracy in CVE-2023-4373| 05-Sep-23
+| 30809 |Fixed inaccuracy in Fireeye path of Openssl| 05-Sep-23
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-vulnerability-management/fixed-reported-inaccuracies.md]:
```suggestion
| 30809 |Fixed inaccuracy in FireEye path of OpenSSL| 05-Sep-23
```

File: microsoft-365/security/defender-vulnerability-management/fixed-reported-inaccuracies.md [hunk 2]
```diff
@@ -39,7 +48,12 @@ Inaccuracy report ID |Description |Fix date |
 |30002 |Fixed inaccuracy in KeePass versions| 10-Aug-23
 | - |Added MDVM support to ODBC and OLEDB| 10-Aug-23
 |29552 |Fixed inaccuracy in Dell Command Update| 14-Aug-23
-
+| - |Fixed inaccuracy in CVE-2021-36234| 22-Aug-23
+| - |Fixed inaccuracy in CVE-2021-36283|	22-Aug-23
+| 30303 | MDVM doesn't currently support Lenovo thinkpad models : E15-gen 4 , L13, L490, T490, T490s, T470s| 29-Aug-23
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-vulnerability-management/fixed-reported-inaccuracies.md]:
```suggestion
| 30303 | MDVM doesn't currently support Lenovo ThinkPad models E15-gen 4, L13, L490, T490, T490s, and T470s| 29-Aug-23
```

File: microsoft-365/security/defender-vulnerability-management/fixed-reported-inaccuracies.md [hunk 3]
```diff
@@ -39,7 +48,12 @@ Inaccuracy report ID |Description |Fix date |
...
+| 29397	| Fixed inaccuracy in Microsoft powershell|	29-Aug-23
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-vulnerability-management/fixed-reported-inaccuracies.md]:
```suggestion
| 29397	| Fixed inaccuracy in Microsoft PowerShell|	29-Aug-23
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-Jagadeeshl, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Comment by siosulli** [comment]
Thanks @yogkumgit - This PR was created in the public repo by mistake and is not ready to be merged. I will actually close this one out and create one in the private repo as we are not sure when the changes will be signed off.

**Comment by siosulli** [comment]
I have created this https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/22265 in the private repo for these changes. Closing out this PR.

**Pull request closed without merging**